engine/src/mblandroidtypeface.cpp

```diff
@@ -87,10 +90,12 @@ void MCAndroidTypefaceRelease(MCAndroidTypefaceRef p_typeface)
 
 bool MCAndroidTypefaceGetMetrics(MCAndroidTypefaceRef p_typeface, uint32_t p_size, float &r_ascent, float &r_descent, float &r_leading, float &r_xheight)
 {
-	bool t_success = true;
-	
+    // Skia APIs expect typefaces to be passed as shared pointers
+    sk_sp<SkTypeface> t_typeface((SkTypeface*)p_typeface);
+    t_typeface->ref();
```

Comment:
looks like there's a convenience function for this - sk_ref_sp(T* obj) which wraps the ptr and increments the ref count